executor/api/rest/middlewares.go

```diff
@@ -1,9 +1,10 @@
 package rest
 
 import (
+	"net/http"
```

Comment:
This happened to just be part of my auto-save in VSCode - it ordered the imports here.